tests/ZfrOAuth2Test/Server/Grant/AuthorizationGrantTest.php

```diff
@@ -244,7 +255,7 @@ public function testCanCreateTokenResponse($hasRefreshGrant)
 
         $response = $this->grant->createTokenResponse($request, new Client(), $owner);
 
-        $body = json_decode($response->getContent(), true);
+        $body = json_decode($response->getBody(), true);
```

Comment:
Hi @weierophinney . It does not seem to work. Here is how I create the response:

```php
 $stream = new Stream('php://temp');
        $stream->write(json_encode(['error' => $exception->getCode(), 'error_description' => $exception->getMessage()]));

        return new Response($stream, 400);
```

then in my tests:

```php
$body     = json_decode($response->getBody(), true);
```

$body is NULL. I've tried getBody()->getContents(), but I retrieve an empty string. How should it work?